### PR TITLE
Next gen

### PR DESCRIPTION
**Pull request by hortega**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
+<a href="https://www.twilio.com">
+  <img src="https://static0.twilio.com/marketing/bundles/marketing/img/logos/wordmark-red.svg" alt="Twilio" width="250" />
+</a>
+
+
 # User Account Verification with Node.js/Express and Twilio
 
-[![Build Status](https://travis-ci.org/TwilioDevEd/warm-transfer-node.svg?branch=master)](https://travis-ci.org/TwilioDevEd/warm-transfer-node)
+[![Build Status](https://travis-ci.org/TwilioDevEd/account-verification-node.svg?branch=master)](https://travis-ci.org/TwilioDevEd/account-verification-node)
```
Comment (acamino) [inline README.md:8]: Thanks! 🎉

File: README.md [hunk 2]
```diff
@@ -18,7 +23,7 @@ This project is build using [Node.js](https://nodejs.org) and depends on [MongoD
 
    ```bash
    $ git clone git@github.com:TwilioDevEd/account-verification-node.git
-   $ cd warm-transfer-node
+   $ cd account-verification-node
```
Comment (acamino) [inline README.md]: One more time, thanks! 😉

File: .travis.yml [hunk 3]
```diff
@@ -11,6 +11,8 @@ env:
     - TWILIO_ACCOUNT_SID=ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
```
Comment (acamino) [inline .travis.yml]: Did you consider removing the env vars from here? Is worthy to note that the only one that is required for the build is `CXX`.

**Comment by acamino** [comment]
Thanks @hortega! LGTM

**Comment by hortega** [comment]
Thanks @acamino !

File: package.json [hunk 4]
```diff
@@ -25,20 +25,21 @@
...
     "body-parser": "^1.12.0",
     "connect-flash": "^0.1.1",
+    "dotenv-safe": "^3.0.0",
     "express": "^4.12.0",
     "express-session": "^1.10.3",
     "jade": "^1.9.2",
     "mongoose": "^4.0.6",
     "morgan": "^1.5.1",
-    "twilio": "^1.11.1"
+    "twilio": ">=3.0.0-rc.16 < 3.1.0"
```
Comment (joliveros) [inline package.json:39]: this is fine. However you could shorten it by using `~3.0.0-rc.16`

**Pull request merged**